infoFileComparison/src/main/scala/za/co/absa/hermes/infoFileComparison/InfoComparisonConfig.scala

```diff
@@ -0,0 +1,86 @@
+package za.co.absa.hermes.infoFileComparison
+
+import scopt.OptionParser
+
+/**
+  * This is a class for configuration provided by the command line parameters
+  *
+  * Note: scopt requires all fields to have default values.
+  *       Even if a field is mandatory it needs a default value.
+  */
+case class InfoComparisonConfig(newPath: String = "",
+                                refPath: String = "",
+                                outPath: String = "")
+
+object InfoComparisonConfig {
+
+  def getCmdLineArguments(args: Array[String]): InfoComparisonConfig = {
+    val parser = new CmdParser("spark-submit [spark options] TestUtils.jar")
+
+    val optionCmd = parser.parse(args, InfoComparisonConfig())
+    if (optionCmd.isEmpty) {
+      // Wrong arguments provided, the message is already displayed
+      System.exit(1)
```

Comment:
Is `System.exit()` really necessary here? Usually it's considered a very bad practice using it anywhere in the code except for the main class of a console app. `System.exit()` immediately and unconditionally terminates the whole VM only leaving a tiny choice of ways to intercept it and retain a control on the program execution.

Reply:
Also, just in case you want to return different status codes and would like to test it, you could take a look at this fixure dedicated for testing return status codes from `System.exit()`
https://github.com/AbsaOSS/spline/blob/develop/commons/src/main/scala/za/co/absa/spline/common/scalatest/SystemExitFixture.scala
and the use cases
- https://github.com/AbsaOSS/spline/blob/develop/commons/src/test/scala/za/co/absa/spline/common/scalatest/SystemExitFixtureSpec_SuiteHook.scala
- https://github.com/AbsaOSS/spline/blob/develop/commons/src/test/scala/za/co/absa/spline/common/scalatest/SystemExitFixtureSpec_Methods.scala

Reply:
Ok, I tried something. Anyway, I am waiting for the new scopt version with better error management.